src/components/ConnectBankAccountButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as ReimbursementAccount from '@userActions/ReimbursementAccount';
+import Button from './Button';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+
+type ConnectBankAccountButtonProps = {
+    policyID: string;
+    style: StyleProp<ViewStyle>;
```

Comment:
I guess it can be optional
```suggestion
    style?: StyleProp<ViewStyle>;
```